### PR TITLE
[ONNX] Remove aten parameter

### PR DESCRIPTION
**Pull request by hwangdeyu**
Remove deprecated export() parameter `aten`.

Cause it's equivalent to setting `operator_export_type=OperatorExportTypes.ONNX_ATEN`.

### CODE REVIEW

File: torch/onnx/utils.py [hunk 1]
```diff
@@ -529,17 +526,12 @@ def _model_to_graph(model, args, verbose=False,
...
+                            export_type=ExportTypes.PROTOBUF_FILE, example_outputs=None,
+                            google_printer=False, opset_version=None, _retain_param_name=True,
                             keep_initializers_as_inputs=None, custom_opsets=None, add_node_names=True,
                             do_constant_folding=True, dynamic_axes=None):
-    if aten:
-        assert operator_export_type is None
-        assert aten
-        operator_export_type = OperatorExportTypes.ONNX_ATEN
-    elif operator_export_type is None:
+    if operator_export_type is None:
```
Comment (garymm) [inline torch/onnx/utils.py:534]: I think it'd be better to just change the default value of the arg to OperatorExportTypes.ONNX (both here and in `export()`).

**Review by garymm — CHANGES_REQUESTED** [review]

File: torch/onnx/utils.py (hunk 1 above)
Comment (hwangdeyu) [reply torch/onnx/utils.py:534]: Make sense. But in export(), we may also check if `PYTORCH_ONNX_CAFFE2_BUNDLE` parameter is needed. https://github.com/pytorch/pytorch/blob/27ae5c5ff0ab2ee1d0fc2b401eb614a75b643abb/torch/onnx/utils.py#L80

File: torch/onnx/__init__.py [hunk 2]
```diff
@@ -116,8 +116,6 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
             input nodes of the graph, in order.
         output_names (list of str, default empty list): names to assign to the
             output nodes of the graph, in order.
-        aten (bool, default False): [DEPRECATED. use operator_export_type] equivalent to
-            setting ``operator_export_type=OperatorExportTypes.ONNX_ATEN``.
         operator_export_type (enum, default OperatorExportTypes.ONNX):
```
Comment (garymm) [inline torch/onnx/__init__.py:119]: Thanks for pointing out that the the behavior depends on PYTORCH_ONNX_CAFFE2_BUNDLE. Please update the documentation to something like:
```
operator_export_type (OperatorExportType enum, default None):
    If None, usually defaults to ONNX.
    If PyTorch was built with ``-DPYTORCH_ONNX_CAFFE2_BUNDLE``, defaults to ONNX_ATEN_FALLBACK.
```

**Review by garymm — CHANGES_REQUESTED** [review]

File: torch/onnx/__init__.py (hunk 2 above)
Comment (hwangdeyu) [reply torch/onnx/__init__.py]: Thanks, added it.

**Review by garymm — APPROVED** [review]

**Pull request merged**